checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java

```diff
@@ -486,12 +452,8 @@ protected void setSelfTypeInInitializationCode(
                     && getUninitializedInvariantFields(store, path, false, Collections.emptyList())
                             .isEmpty()) {
                 if (classType.isFinal()) {
-                    annotation = COMMITTED;
-                } else if (useFbc) {
-                    annotation = createFreeAnnotation(classType);
-                } else {
-                    annotation = createUnclassifiedAnnotation(classType);
-                }
+                    annotation = INITIALIZED;
+                } else annotation = createFreeAnnotation(classType);
```

Comment:
Nit: for symmetry with the `then` branch I would put this statement also into a block.